source/nodes.property_extractor.tidally_truncated_NFW_fit.F90

```diff
@@ -190,8 +192,18 @@ function tidallyTruncatedNFWFitExtract(self,node,time,instance)
        radiusScale  =  darkMatterProfile                     %scale              (                                              )
        radiusVirial =  self             %darkMatterHaloScale_%radiusVirial       (node                                          )
        ! Choose radii for fitting.
-       radiusMaximum=    fractionRadiusOuter*radiusOuter
-       radiusMinimum=min(fractionRadiusScale*radiusScale,fractionMaximum*radiusMaximum)
+       radiusMaximum=radiusVirial
+       if (radiusOuter > radiusVirial) then
+           radiusMaximumCountRadii=int(log10(radiusMaximumScaleVirialMaximum)*dble(radiusMaximumCountRadiiPerDecade)+1.0d0)
+           factorStepRadius       =    log10(radiusMaximumScaleVirialMaximum)/dble(radiusMaximumCountRadii         ) 
+           do i=1,radiusMaximumCountRadii
```

Comment:
This was previously:
```
         do n=1,radiusMaxiumCountRadii
             radiusMaximum                     =10**(log10(radiusVirial)+step*dble(i))
```
So the loop variable was `n` but on the following line it used `i` to increment `radiusMaximum`. Most likely `i=0` initially, so this would not have actually increased `radiusMaximum`, leaving it always equal to `radiusVirial`. I've corrected this (just using `i` as the loop variable), but it would be good if you can check if this still gives good results for the fits.

Reply:
I recompiled and re-ran the parameter with the bug fixed. The fits look good.